cluster-autoscaler/cloudprovider/oci/README.md

```diff
@@ -122,6 +140,10 @@ Configuration via environment-variables:
 - `OCI_REFRESH_INTERVAL` - Optional refresh interval to sync internal cache with OCI API defaults to `2m`.
 - `OCI_USE_NON_POOL_MEMBER_ANNOTATION` - Optional. If true, the node will be annotated as non-pool-member if it doesn't belong to any instance pool and the time-consuming instance pool lookup will be skipped.
```

Comment:
Probably want to note that `OCI_USE_NON_POOL_MEMBER_ANNOTATION` applies to instance pools only.